src/sentry/workflow_engine/migration_helpers/alert_rule.py

```diff
@@ -40,6 +40,15 @@
     "team_id": "owner_team_id",
 }
 
+PRIORITY_MAP = {
+    "warning": DetectorPriorityLevel.MEDIUM,
+    "critical": DetectorPriorityLevel.HIGH,
+}
+
+
+class MissingACITableException(Exception):
+    pass
```

Comment:
this isn't actually doing anything when it's raised - do you want to log and then raise?

Reply:
jfyi - this still does stuff if it's raise. it will halt execution and stop the migration (super helpful with control flows). this is actually a solid example of when / how to use exceptions to help with control flow, if we have a malformed attribute we can throw this and find it.